import java.io.*;
//case 3
/*class Z{
	public static void main(String[] args)throws FileNotFoundException{

    }
}*/

//case 2
/*class Z{
	public static void main(String[] args){
		try{
              
		}catch(ArithmeticException e){

		}
    }
}*/

//case 1
/*class Z{
	public static void main(String[] args){
		try{
              
		}catch(FileNotFoundException e){

		}
    }
}*/
/*if there is  not any problem creating checked exception in tryblock and u make its approriate catch block than it gives compile 
time error but there is nothing like that in case of unchecked exception*/
//in exception declaration there is no problem in both the cases.